techniques/system/common/1.0/cf-served.st

```diff
@@ -98,15 +98,12 @@ bundle common def
         string  => translatepath("${sys.workdir}/policy_server.dat"),
         comment => "Path to file containing address to policy server";
 
-   # Use localhost as default policy_server if no policy_server.dat exists
-    policy_server_set::
+      # Use "rudder" as default policy_server if no policy_server.dat exists
       "policy_server"
-        string  => readfile("${policy_server_file}", 2048),
+        string  => ifelse(fileexists("${def.policy_server_file}"),
+                          readfile("${policy_server_file}", 2048),
```

Comment:
No good reason. Both work since it's a local variable, but that is confusing. I'll fix it.